README.md

```diff
@@ -111,34 +86,33 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
```

Comment:
- [ ] As suggested [here](https://github.com/stino-x/HELLO-NEW/pull/2#pullrequestreview-1247915815), please consider removing this comment. It is a mere template as guide on how to fill your information in.